ArduCopter/system.cpp

```diff
@@ -79,6 +79,9 @@ void Copter::init_ardupilot()
 
     // initialise notify system
     notify.init();
+
+    //Setting the mode to INITIALISING
```

Comment:
This should be above `notify.init()`

Also, while the comment you've added is certainly in flavour - I'm thinking we should kill most of them as redundant in some future PR.

Reply:
Made a new mode for the copter and initialized control_mode with the INITIALISING mode due to which I do not need this line anymore